.github/drafts/15938/15938_Audit_Logs_Outline.md

```diff
@@ -0,0 +1,108 @@
+# Audit logs
+
+Audit logs are a vital way to keep records of user activity related to app changes, platform settings, and sensitive data. With access to this information, you can ensure that you're able to easily follow up and address any unexpected or undesirable events.
+
+{Screenshot of audit logs page}
+
+---
+
+## Logged events
+
+(Should we include the technical names of the events? Or maybe one example event per type? i.e. `datasource.created` etc.)
+
+There are several types of events tracked in Appsmith's audit logs:
+
+- App-specific activity
+    - Changes to pages/widgets (`page.updated`)
+    - Datasource configuration(`datasource.updated`)
+    - Running queries(`query.executed`)
+    - Deleting apps
+
+- Admin settings changes
+    - Who changed certain settings (`instance_setting.updated`)
+    - Changes/new groups or roles (`group.updated`)
+
+- Sensitive information access
+    - Which users have accessed user data
+    - Which users have accessed billing data
+
+- User sign-up and logins
+    - New users
+    - Unsuccessful logins
+
+## Event details
+
+Each logged item has a number of data points that describe that event, as well as any related apps, datasources, queries, and workspaces.
+
+| Data | Description | Details |
+|------|-------------|---------|
+| event | Each action performed on Appsmith app(edit or view mode) is classified as event |  |
+| timestamp | Displays the date and time of a logged event. |  |
+| user | Displays the user who made performed this event | `id`, `email`, `name`, `ipAddress` |
+| resource | The type of resource on which this event was performed (Query, Datasource, etc.) | `id`, `type`, `name` |
+| app | Application on which the action is performed. | `id`, `name`, `git` {`branch`, `default`} |
+| workspace | Workspace on which the action is performed. | `id`, `name` |
+| metadata | Platform details, such as the Appsmith version. | `ipAddress`, `appsmithVersion`, `createdAt` |
+
+Below is an example of a log item:
+
+```JSON
+{
+    "event": "datasource.created",
+    "timestamp": "2022-06-29T08:36:33.507+00:00",
+    "user": {
+        "id": "62bc0f11545c4c00b19d5c65",
+        "email": "john@appsmith.com",
+        "name": "John Doe",
+        "ipAddress": "223.183.39.7"
+    },
+    "resource": {
+        "id": "62bc0f11545c4c00b19d5c59",
+        "type": "Datasource",
+        "name": "Movies"
+    },
+    "app": {
+        "id": "62bc0f11545c4c00b19d5c68",
+        "name": "Standup App",
+        "git": {
+            "branch": "feat/new-ui",
+            "default": "master"
+        }
+    },
+    "workspace": {
+        "id": "62bc0f11545c4c00b19d5c64",
+        "name": "Appsmith Internal Apps"
+    },
+    "metadata": {
+        "ipAddress": "223.183.39.7",
+        "appsmithVersion": "1.7.5",
+        "createdAt": "2022-06-29T08:36:33.507+00:00"
+    }
+}
+```
+
```

Comment:
Here's where we should give a detailed list of all the event types and their description as shared in the Events list notion doc

Reply:
Sure, I'll build that in. [Link](https://www.notion.so/appsmith/Events-List-bcaba08d34f94b90a655d835de6788b7) to the notion doc for easy access later.

Reply:
Added to the existing "Logged Events" heading, I think it will fit nicely there!